frontend/views/pages/PayGroup.vue

```diff
@@ -3,157 +3,143 @@ page(
   pageTestName='payGroupPage'
   pageTestHeaderName='payGroupTitle'
 )
-  template(#title='') {{ L('Pay Group') }}
+  template(#title='') {{ L('Payments') }}
 
   template(#sidebar='' v-if='hasPayments')
-    .c-summary-item(
-      v-for='(item, index) in paymentSummary'
-      :key='index'
+    month-overview
+
+  i18n.p-descritpion.has-text-1(
+    tag='p'
+    data-test='openIncomeDetailsModal'
+    @click='openModal("IncomeDetails")'
+    :args='{ receive_or_send: `<strong>${needsIncome ? L("receiving") : L("sending")}</strong>`, r1: `<button class="link js-btnInvite">`, r2: "</button>"}'
+  ) You are currently {receive_or_send} mincome. You can change this at any time by updating your {r1}income details{r2}.
```

Comment:
Hmm, this is a tricky sentence to localize. The issue here is that I don't think every language will be able to support this sort of translation style. In the first sentence, there are two separate translations goes on, that are being combined together:

1. `You are currently {receive_or_send} mincome.`
2. The words `receiving` and `sending`, translated individually, and being placed into the first sentence.

The issue is that in some languages, it's possible that "receiving" might get translated one way by itself, but in the context of the full sentence, it might need to be translated another way.

So I recommend using a function to return one of two sentences for the first sentence:

1. `You are currently receiving mincome.`
2. `You are currently sending mincome.`

One way of doing this is to define a computed property that returns one of those sentences, and then insert it as a variable here.